tests/unit/dataactbroker/test_file_routes.py

```diff
@@ -45,23 +48,58 @@ def test_is_period(file_app, database):
 
     curr_date = datetime.now()
     diff = timedelta(days=1)
-    gtas_current = BannerFactory(start_date=curr_date - diff, end_date=curr_date + diff, block_certification=False,
-                                 message='third', application_type=application)
-    database.session.add(gtas_current)
+    third_current = BannerFactory(start_date=curr_date - diff, end_date=curr_date + diff, block_certification=False,
+                                  message='third', application_type=fabs_app, banner_type="info")
+    database.session.add(third_current)
 
     response = file_app.get("/v1/list_banners/")
     response_json = json.loads(response.data.decode('UTF-8'))
-    assert response_json['data'][0]['notice_block'] is False
+    third_response = {
+        "start_date": str(curr_date - diff),
+        "end_date": str(curr_date + diff),
+        "header": None,
+        "message": "third",
+        "type": fabs_app.application_name,
+        "banner_type": "info",
+        "notice_block": False
+    }
+    assert response_json['data'][0] == third_response
 
     curr_date = datetime.now()
     diff = timedelta(days=1)
-    gtas_current = BannerFactory(start_date=curr_date - diff, end_date=curr_date + diff, block_certification=True,
-                                 message='fourth', application_type=application)
-    database.session.add(gtas_current)
+    fourth_current = BannerFactory(start_date=curr_date - diff, end_date=curr_date + diff, block_certification=True,
+                                   message='fourth', application_type=dabs_app, banner_type="warning")
+    fifth_current = BannerFactory(start_date=curr_date - diff, end_date=curr_date + diff, block_certification=True,
+                                  message='fifth', application_type=login_app, banner_type="info", header='FIFTH')
+    database.session.add_all([fourth_current, fifth_current])
 
     response = file_app.get("/v1/list_banners/")
     response_json = json.loads(response.data.decode('UTF-8'))
-    assert response_json['data'][1]['notice_block'] is True
+    fourth_response = {
+        "start_date": str(curr_date - diff),
+        "end_date": str(curr_date + diff),
+        "header": None,
+        "message": "fourth",
+        "type": dabs_app.application_name,
+        "banner_type": "warning",
+        "notice_block": True
+    }
+    sort_key = (lambda k: k['message'])
+    assert sorted(response_json['data'], key=sort_key) == sorted([third_response, fourth_response], key=sort_key)
+
+    # Adding another check for just the login banner, which was ignored in the last call
```

Comment:
Since we're adding several more tests, it might be a good idea to make comments on the existing ones to denote what they're testing. This particular comment sounds like a commit message, not a comment. `adding another check` sounds like what you did, not a comment on the code itself. Just `Check for only login banner` or something similar would be plenty